src/test/java/com/github/lombrozo/testnames/rules/ml/ModelSourceFileSystemTest.java

```diff
@@ -25,27 +25,41 @@
 
 import java.io.IOException;
 import java.nio.file.Path;
+import java.util.concurrent.atomic.AtomicReference;
+import opennlp.tools.postag.POSModel;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
 /**
  * Tests for {@link ModelSourceFileSystem}.
  * @since 0.10
- * @todo #1:90min Enable ML tests when caching will be implemented.
- *  For not tests related to ML take lots of time and it's hard to develop in that mode.
- *  We have to implement a caching or increase the speed of tests in order to use them in
- *  every-day development.
  */
 @Disabled
 final class ModelSourceFileSystemTest {
 
+    /**
+     * Cache.
+     */
+    private static final AtomicReference<POSModel> CACHE = new AtomicReference<>();
+
+    @BeforeAll
+    static void setUp() throws IOException {
```

Comment:
@h1alexbel As I have understood your idea, you are trying to warm-up the cache before the usage in all the tests. I believe you don't need to do this. The first call to the `mode.model()` in the first test should do all the work. The rest tests will reuse cached model.

(At least, it must be so. If I'm wrong, please, correct me.)